extensions/devservices/keycloak/pom.xml

```diff
@@ -34,7 +34,7 @@
         </dependency>
         <dependency>
             <groupId>org.keycloak</groupId>
-            <artifactId>keycloak-core</artifactId>
+            <artifactId>keycloak-admin-client</artifactId>
```

Comment:
This is problematic IMO as it drags RESTEasy components in anything using the Keycloak Dev Services. Is it actually needed to depend on the admin client?

And that probably explain the mixed stacks issue in `NamedOidcClientFilterDevModeTest`.

Reply:
@gsmet Thanks Guillaume, indeed, here it is not required

Reply:
@pskopek Looks like `keycloak-core` has to be released independently and keycloak client will depend on it.

Reply:
@sberyozkin no, keycloak-core cannot be released independently. This will load a lot of other dependencies which we wanted to get rid of. I am looking for a solution to fix the issue.

Reply:
@gsmet @sberyozkin  the issue is fixed now. Dependency on `keycloak-admin-client` is correct, the resteasy should be excluded as it is not needed here. (The classes from `keycloak-core` formerly used in `quarkus-devservices-keycloak` are part of `keycloak-admin-client` now and all Keycloak external projects should us it.)